shell/pages/c/_cluster/auth/user.retention/index.vue

```diff
@@ -0,0 +1,381 @@
+    ids.forEach((key) => {
+      settings[key].value = userRetentionSettings[key];
+
+      if (key === SETTING.USER_LAST_LOGIN_DEFAULT && userRetentionSettings[key]) {
+        settings[key].value = dayjs(Number(userRetentionSettings[key])).format();
+      }
+    });
+
+    await Promise.all(ids.map((setting) => settings[setting].save()));
+
+    btnCB(true);
+    store.dispatch(
+      'growl/success',
+      {
+        title:   'Save user retention settings',
+        message: 'User retention settings have been updated successfully',
+      },
+      { root: true }
+    );
+    routeBack();
+  } catch (err) {
+    error.value = (err as Error).message;
+    btnCB(false);
+  }
+};
+
+const router = useRouter();
+const routeBack = () => {
+  router.back();
+};
+
+onBeforeRouteUpdate((_to: unknown, _from: unknown) => {
+  if (!isAdminUser(store.getters)) {
+    router.replace({ name: 'home' });
+  }
+});
+</script>
+
+<template>
+  <div>
+    <user-retention-header />
+    <div
+      v-if="!loading"
+      class="form-user-retention"
+    >
+      <banner
+        v-if="error"
+        color="error"
+      >
+        {{ error }}
+      </banner>
+      <div class="input-fieldset">
+        <checkbox
+          v-model="disableAfterPeriod"
+          data-testid="disableAfterPeriod"
+          :label="t('user.retention.edit.form.disableAfter.checkbox')"
+        />
+        <labeled-input
+          v-model="userRetentionSettings[SETTING.DISABLE_INACTIVE_USER_AFTER]"
+          data-testid="disableAfterPeriodInput"
+          :tooltip="t('user.retention.edit.form.disableAfter.input.tooltip')"
+          class="input-field"
+          :label="t('user.retention.edit.form.disableAfter.input.label')"
+          :disabled="!disableAfterPeriod"
+        />
+      </div>
+      <div class="input-fieldset">
+        <checkbox
+          v-model="deleteAfterPeriod"
+          data-testid="deleteAfterPeriod"
+          :label="t('user.retention.edit.form.deleteAfter.checkbox')"
+        />
+        <labeled-input
+          v-model="userRetentionSettings[SETTING.DELETE_INACTIVE_USER_AFTER]"
+          data-testid="deleteAfterPeriodInput"
+          :tooltip="t('user.retention.edit.form.deleteAfter.input.tooltip')"
+          class="input-field"
+          :label="t('user.retention.edit.form.deleteAfter.input.label')"
+          :sub-label="t('user.retention.edit.form.deleteAfter.input.subLabel')"
+          :disabled="!deleteAfterPeriod"
+        />
+      </div>
+      <template
+        v-if="disableAfterPeriod || deleteAfterPeriod"
+      >
+        <div class="input-fieldset pt-12">
+          <labeled-input
+            v-model="userRetentionSettings[SETTING.USER_RETENTION_CRON]"
+            data-testid="userRetentionCron"
+            class="input-field"
+            required
+            type="cron"
+            :label="t('user.retention.edit.form.cron.label')"
+            :sub-label="t('user.retention.edit.form.cron.subLabel')"
```

Comment:
This area starts to feel pretty text-heavy with the multiline sub-label and cron hint. Could this input use a cron expression placeholder + input validator commented above instead of the sub-label?

Reply:
Agreed, and I think the suggestion will clean this up but will change the design in a minor way. @edenhernandez-suse what do you think?